.github/workflows/sweep-check.yaml

```diff
@@ -0,0 +1,33 @@
+name: Sweepers Check
```

Comment:
please integrate this into an existing `lint.yaml` workflow

Reply:
ok